tests/mocks.py

```diff
@@ -1146,7 +1174,11 @@ def to_response(i):
         items = list(self._transactions)
 
         if objectIdentifier:
-            items = [i for i in items if i["objectPHID"] == objectIdentifier]
+            items = [
+                i
+                for i in items
+                if i["objectPHID"] == objectIdentifier or i["id"] == objectIdentifier
+            ]
```

Comment:
This is not how the Phabricator API behaves, why are you making this change?

Reply:
That's a bug.  Searching by `objectIdentifier` does a fuzzy match on PHID or monogram.  There is no search by ID.